two-factor.php

```diff
@@ -236,3 +236,74 @@ function wpcom_vip_two_factor_admin_notice() {
 	</div>
 	<?php
 }
+
+// Filter `woocommerce_prevent_admin_access` to allow users to set up 2FA on sites running WooCommerce.
+add_filter( 'woocommerce_prevent_admin_access', 'wpcom_vip_should_woocommerce_prevent_admin_access', 10, 1 );
+
+function wpcom_vip_should_woocommerce_prevent_admin_access( $prevent_access ) {
+
+	// If the 2FA plugin won't be enabled or $prevent access is already `false`, do nothing and return early.
+	$vip_two_factor_enabled = apply_filters( 'wpcom_vip_enable_two_factor', true );
+	if ( true !== $vip_two_factor_enabled || ! $prevent_access ) {
+		return $prevent_access;
```

Comment:
Minor: Return false rather than return $prevent_access for clarity of intention.
```suggestion
		return false;
```